pkgs/development/python-modules/heatshrink2/default.nix

```diff
@@ -0,0 +1,22 @@
+{ buildPythonPackage, fetchPypi, lib }:
+
+let
+  pname = "heatshrink2";
+  version = "0.11.0";
+
+in
+buildPythonPackage {
+  inherit pname version;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-xihtHUuS+664Ow/YtfgxyTUNzRBG53blqqg9G+Q4Nfc=";
+  };
+
```

Comment:
There are tests https://github.com/eerimoq/pyheatshrink/tree/master/tests . Are those being executed? Regardless of that we should add a pythonImportsCheck.